src/topp/QualityControl.cpp

```diff
@@ -143,6 +144,9 @@ class TOPPQualityControl : public TOPPBase
     setValidFormats_("in_trafo", {"trafoXML"});
     registerTOPPSubsection_("MS2_id_rate", "MS2 ID Rate settings");
     registerFlag_("MS2_id_rate:assume_all_target", "Forces the metric to run even if target/decoy annotation is missing (accepts all pep_ids as target hits).", false);
+    registerStringOption_("out_evd", "<Path>", "", "EvidenceTXT with QC information", false);
```

Comment:
maybe a more telling paramter and decscription. Just looking at "out_evd" I would have no initial clue what it does